src/components/CommentsList.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext, useEffect, useState } from 'react';
+import { CommentItem } from './CommentItem';
+import { getComments } from '../api/comments';
+import { ListContext } from './ListContext';
+import { Loader } from './Loader';
+import { WriteCommentButton } from './WriteCommmentButton';
+
+export const CommentsList: React.FC = () => {
+  const {
+    selectedPost,
+    comments,
+    setComments,
+    isErrorComment,
+    setIsErrorComment,
+  } = useContext(ListContext);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsErrorComment(false);
+
+    getComments(selectedPost.id).then(commentsFromServer => {
+      setComments(commentsFromServer);
+      setIsLoading(false);
+    })
+      .catch(() => {
+        setIsLoading(false);
+        setIsErrorComment(true);
+        setComments([]);
+      });
+  }, [selectedPost]);
```

Comment:
<img width="718" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/70220667/50781209-7a89-4be0-9580-2effba1128ab">
try to fix that